lib/spark_api/configuration.rb

```diff
@@ -44,6 +44,7 @@ module Configuration
     DEFAULT_COMPRESS = false
     DEFAULT_TIMEOUT = 5 # seconds
     DEFAULT_MIDDLEWARE = 'spark_api'
+    DEFAULT_DICTIONARY_VERSION = nil
```

Comment:
there is no default version?

Reply:
Not through this client. Not specifying the version in the URI will yield the default.